plugins/linguist-backend/api-report.md

```diff
@@ -62,12 +54,16 @@ export class LinguistBackendDatabase implements LinguistBackendStore {
 
 // @public (undocumented)
 export interface LinguistBackendStore {
+  // (undocumented)
```

Comment:
Isn't an `LinguistBackendStore` internal concern for the backend given that users only pass in the API(client)? Perhaps it could be unexported as well?

Reply:
Yes, that does not need to be exported. I've removed it 👍